<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QCheckBox" name="invertCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>invert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3" colspan="3">
    <layout class="QGridLayout" name="imageGridLayout"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="resetPushButton">
     <property name="text">
      <string>reset previewimage</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="radiusSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string>radius </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="blurCheckBox">
     <property name="text">
      <string>blur</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="heightSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="prefix">
      <string>height </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QSpinBox" name="posXSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="prefix">
      <string>posX </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="posYSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="prefix">
      <string>posY </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="widthSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="prefix">
      <string>width </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>You can drop custom image here:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>blurCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>invertCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>posXSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>radiusSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>posYSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>heightSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
